gradle/project.gradle.kts

```diff
@@ -69,14 +70,10 @@ dependencies {
     // Kotlin
     compile("org.jetbrains.kotlin", "kotlin-stdlib-jdk8", kotlinVersion)
     compile("org.jetbrains.kotlin", "kotlin-reflect", kotlinVersion)
-    compile("org.jetbrains.kotlin", "kotlin-test", kotlinVersion)
-
-    // Java Native
-    compile("net.java.dev.jna", "jna-platform", "4.5.1")
 
     // Logging
-    compile("org.slf4j", "slf4j-api", "1.7.25")
-    compile("ch.qos.logback", "logback-classic", "1.2.3")
+    compile("org.slf4j", "slf4j-api", "1.8.0-beta1")
+    compile("ch.qos.logback", "logback-classic", "1.3.0-alpha4")
```

Comment:
Not sure alpha/beta version are ok. I advise using latest release.

Reply:
well these are the latest releases. If they are on maven, they are surely fairly stable, and you also use alpha versions of twelvemonkeys ;)

Reply:
beta is not release. It's on maven for the people who want to live on edge. We are not those people anymore. Exceptions (12monkey) apply when an important bug has been fixed. 

Only stable releases unless necessary pls.